docs/developer/unsupported_server_args.md

```diff
@@ -0,0 +1,76 @@
+<h1 align="center">Unsupported server args override for Velero deployment and Node-agent Daemon set containers</h1>
+
+### Prerequisites
+- OpenShift CLI
+- OADP Operator installed on OpenShift cluster
+- OADP versions 1.3 and above
+
+
+### Unsupported server args override for Velero deployment container
+
+- Create a ConfigMap with key-value pairs, the keys correspond to the velero server arguments and values correspond to the argument value. Sample configmap is as follows:
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: my-config-velero-server
+  namespace: openshift-adp
+data:
+  # Configuration values
+  features: "EnableCSI"
+  uploader-type: "kopia"
+  fs-backup-timeout: "10h"
+```
+- Now create/update the DPA with the annotation `oadp.openshift.io/unsupported-velero-server-args=<ConfigMap-Name>`. Considering our example the annotation would be:
+```yaml
+  annotations:
+    oadp.openshift.io/unsupported-velero-server-args: my-config-velero-server
+```
+- Finally, once the DPA is successfully reconciled then verify the velero deployment container args and check whether they match with the ones specified in the configmap or not. For our example, it would look like:
```

Comment:
Is there any command to do so?

I am thinking here of:
```shell
$ oc get pod -n openshift-adp -l component=velero -l deploy=velero -o json | jq '.items[].spec.containers[].args'
[
  "server",
  "--uploader-type=restic",
  "--fs-backup-timeout=4h",
  "--restore-resource-priorities=securitycontextconstraints,customresourcedefinitions,klusterletconfigs.config.open-cluster-management.io,managedcluster.cluster.open-cluster-management.io,namespaces,roles,rolebindings,clusterrolebindings,klusterletaddonconfig.agent.open-cluster-management.io,managedclusteraddon.addon.open-cluster-management.io,storageclasses,volumesnapshotclass.snapshot.storage.k8s.io,volumesnapshotcontents.snapshot.storage.k8s.io,volumesnapshots.snapshot.storage.k8s.io,datauploads.velero.io,persistentvolumes,persistentvolumeclaims,serviceaccounts,secrets,configmaps,limitranges,pods,replicasets.apps,clusterclasses.cluster.x-k8s.io,endpoints,services,-,clusterbootstraps.run.tanzu.vmware.com,clusters.cluster.x-k8s.io,clusterresourcesets.addons.cluster.x-k8s.io",
  "--disable-informer-cache=false"
]

```

Or
```shell
$ oc get pod -n openshift-adp -l component=velero -l name=node-agent -o json | jq '.items[].spec.containers[].args'
[
  "node-agent",
  "server"
]
[
  "node-agent",
  "server"
]
[
  "node-agent",
  "server"
]

```

Reply:
if we can remove `jq` command I think it is a good idea